scss/general.scss

```diff
@@ -4,5 +4,5 @@
 @import "./sections/_hero.scss";
```

Comment:
move this imports to index.scss
general file should contains only general styles for common elements.